src/components/PupilComponents/QuizResultShortAnswer/QuizResultShortAnswer.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import {
+  OakFlex,
+  OakQuizResultItem,
+  OakSpan,
+} from "@oaknational/oak-components";
+import { MathJaxWrap } from "@/browser-lib/mathjax/MathJaxWrap";
+
+export type QuizResultShortAnswerProps = {
+  answers: string | (string | undefined)[];
+  pupilAnswer: string;
+};
+
+export const QuizResultShortAnswer = ({
+  answers,
+  pupilAnswer,
+}: QuizResultShortAnswerProps) => {
+  const feedbackState = (() => {
+    switch (true) {
+      case answers.includes(pupilAnswer):
```

Comment:
I think it would be better to check that the type is array first before running this check